pkg/assembler/clients/operations/package.graphql

```diff
@@ -38,3 +38,49 @@ query Packages($filter: PkgSpec!) {
     ...AllPkgTree
   }
 }
+
+query PackageTypes($filter: PkgSpec!) {
+  packages(pkgSpec: $filter){
+    type
+  }
+}
+
+query PackageNamespaces($filter: PkgSpec!) {
+  packages(pkgSpec: $filter){
+    type
+    namespaces {
+        namespace
+    }
+  }
+}
+
+query PackageNames($filter: PkgSpec!) {
+  packages(pkgSpec: $filter){
+    type
```

Comment:
We can return `IDs`  here just to be consistent with the output from the others see the other fragments defined here: https://github.com/guacsec/guac/blob/main/pkg/assembler/clients/operations/trees.graphql